### PR TITLE
Ux UI fix

### PR DESCRIPTION
**Pull request by ScienceMaster7**
What I did:

- Creating a Footer component with a house Icon that links to home.
- Addition of the Footer component to all pages except for the home page.
- remove the link from the heading.

### CODE REVIEW

File: src/components/Footer.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  justify-content: center;
+  background-color: var(--footer-color);
+  width: 100%;
+  padding: 10px;
+}
+
+.homeIcon {
+  color: var(--sencodary-text-color);
+  font-size: 2rem;
+}
```
Comment (namirsab) [inline src/components/Footer.css:14]: Just a suggestion. I'd name the classes for a component `Footer`  (with the same name of the component)
And use BEM for the children, for instance `Footer__homeIcon`

File: src/pages/Home.js [hunk 2]
```diff
@@ -17,10 +17,10 @@ export default function Home() {
       return renderedhabbits;
     } else {
       return (
-        <h2>
+        <p className="infoFirstVisit">
```
Comment (namirsab) [inline src/pages/Home.js:20]: This class, more than `infoFirstVisit` is for the "empty state", because the user can get to this view again by removing all the skills from the page

File: src/pages/Home.css [hunk 3]
```diff
@@ -17,3 +17,9 @@
   margin: 1rem;
   font-size: 20px;
 }
+
+.infoFirstVisit {
+  align-self: center;
+  margin: 1rem;
```
Comment (namirsab) [inline src/pages/Home.css:23]: If you want the margin to be set depending on the font-size of this element, you should use `em`instead o `rem`

File: src/components/Footer.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { Link } from "react-router-dom";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faHome } from "@fortawesome/free-solid-svg-icons";
+import "./Footer.css";
+import "./colors.css";
```
Comment (namirsab) [inline src/components/Footer.js:5]: I see you are importing `colors.css` everywhere. You could just import them once in  your `index.js` and then you wouldn't need to keep importing them in every component

File: src/components/Footer.css [hunk 5]
```diff
@@ -0,0 +1,14 @@
+.footer {
+  display: flex;
+  position: fixed;
+  bottom: 0;
+  justify-content: center;
+  background-color: var(--footer-color);
+  width: 100%;
+  padding: 10px;
```
Comment (namirsab) [inline src/components/Footer.css:8]: I wouldn't use pixels for paddings here. Some info:
https://daveredfern.com/use-units-css/

TLDR:

- padding is related to the element font-size? -> em
- padding is not related to the element font-size -> rem

**Review by namirsab — CHANGES_REQUESTED** [review]
@ScienceMaster7 good job. Just a few comments. 

Regarding your UX, I wonder if you could simplify it, having always the bottom navigation visible and removing the need of having a burger menu on the top left corner. i think having both can be confusing some times.

image

File: src/components/Footer.css (hunk 5 above)
Comment (mpagels) [reply src/components/Footer.css:8]: >     * padding is related to the element font-size? -> em
> 
>     * padding is not related to the element font-size -> rem
Nice, I will save this for myself. 😉

Comment (ScienceMaster7) [reply src/components/Footer.css:8]: thanks :)
